sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/Utility.java

```diff
@@ -388,86 +399,89 @@ public static ExtractKeyPhrasesResultCollection toExtractKeyPhrasesResultCollect
-                        // TODO: bidirectional relation is gone, do it in another PR
-                        // issue: https://github.com/Azure/azure-sdk-for-java/issues/19180
-//                        final HealthcareEntity targetEntity =
-//                            healthcareEntities.get(getHealthcareEntityIndex(healthcareRelation.getTarget()));
-//                        final HealthcareEntity sourceEntity =
-//                            healthcareEntities.get(getHealthcareEntityIndex(healthcareRelation.getSource()));
-//                        final HealthcareEntityRelationType relationType =
-//                            HealthcareEntityRelationType.fromString(healthcareRelation.getRelationType());
-
-//                        final Map<HealthcareEntity, HealthcareEntityRelationType> targetRelatedEntity =
-//                            entityRelationMap.getOrDefault(targetEntity, new HashMap<>());
-//                        targetRelatedEntity.putIfAbsent(sourceEntity, relationType);
-//                        entityRelationMap.putIfAbsent(targetEntity, targetRelatedEntity);
-//
-//                        if (healthcareRelation.isBidirectional()) {
-//                            final Map<HealthcareEntity, HealthcareEntityRelationType> sourceRelatedEntity =
-//                                entityRelationMap.getOrDefault(sourceEntity, new HashMap<>());
-//                            sourceRelatedEntity.putIfAbsent(targetEntity, relationType);
-//                            entityRelationMap.putIfAbsent(sourceEntity, sourceRelatedEntity);
-//                        }
-                    });
-                }
-                healthcareEntities.forEach(healthcareEntity -> {
-                    if (entityRelationMap.containsKey(healthcareEntity)) {
-                        HealthcareEntityPropertiesHelper.setRelatedEntities(healthcareEntity,
-                            entityRelationMap.get(healthcareEntity));
-                    }
-                });
-
-                final AnalyzeHealthcareEntitiesResult analyzeHealthcareEntitiesResult =
-                    new AnalyzeHealthcareEntitiesResult(
-                        documentEntities.getId(),
-                        documentEntities.getStatistics() == null ? null
-                            : toTextDocumentStatistics(documentEntities.getStatistics()),
-                        null);
                 AnalyzeHealthcareEntitiesResultPropertiesHelper.setEntities(analyzeHealthcareEntitiesResult,
                     IterableStream.of(healthcareEntities));
-                AnalyzeHealthcareEntitiesResultPropertiesHelper.setWarnings(analyzeHealthcareEntitiesResult,
-                    IterableStream.of(warnings));
+
+                // Healthcare Entity relations
+                final List<HealthcareEntityRelation> healthcareEntityRelations =
+                    documentEntities.getRelations().stream().map(
```

Comment:
can `getRelations()` result in null?

Reply:
it can't be null. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json#L1757